src/components/NavBar/AccountMenu.tsx

```diff
@@ -10,15 +10,31 @@ import LogoutIcon from "@mui/icons-material/Logout";
 import ArrowDropDownIcon from "@mui/icons-material/ArrowDropDown";
 
 const styles = {
+  arrow: {
+    color: "white.main",
+    // Similar to MUI box shadow
+    filter:
+      "drop-shadow(0px 5px 5px rgba(0, 0, 0, 0.2)) drop-shadow(0px 8px 10px rgba(0,0,0,0.14)) drop-shadow(0px 3px 14px rgba(0,0,0,0.12))",
+    left: "50%",
+    position: "absolute",
+    top: "100%",
+    transform: "translate(-50%, -45%)",
+  },
   button: {
+    ":hover": {
+      backgroundColor: "primary.light",
+    },
     marginTop: "auto",
-    marginBottom: 2,
+    marginBottom: 1,
+    padding: 1.5,
+    transitionDuration: "0.25s",
   },
   nameContainer: { paddingLeft: 2, textAlign: "left" },
   popover: {
     paddingTop: "10px",
     paddingBottom: "10px",
     borderRadius: "10px",
+    overflow: "visible",
   },
   popoverButton: {
```

Comment:
can we add `borderRadius: 0,` to popoverButton? makes the button look cleaner i think